cluster_loader/README.md

```diff
@@ -0,0 +1,42 @@
+# Cluster Loader
+
+To run [cluster loader](https://github.com/openshift/origin/blob/master/test/extended/cluster/) with snafu following environment variables are necessary:
+
+*KUBECONFIG* and point to kubernetes cluster you want to run cluster loader against
+
+Note: oc binary is also required as cluster loader uses openshift client
+
+Additional environment variables:
+
+*VIPERCONFIG* if you'd like to use custom configuration for running cluster loader.
+
+Please read more about how you'll need to build your custom configuration file at
+[openshift docs](https://docs.openshift.com/container-platform/4.2/scalability_and_performance/using-cluster-loader.html)
+
+
+Note: To index data into elasticsearch you'll also need to set the environment vars *es*,*es_port* and *es_index*
+as well as additional identifers such as *uuid*, *clustername* and *test_user*
+
+## Invoking cluster loader through snafu:
+
+Note: Please build the binary openshift-tests by following
+[docs for installing prereqs](https://github.com/openshift/origin/blob/release-4.3/CONTRIBUTING.adoc#installing-prerequisites)  and
+also [building binary](https://github.com/openshift/origin/blob/master/HACKING.md#end-to-end-e2e-and-extended-tests)
+
+Note: you'll also need to install following python packages as follows:
+
+```bash
+pip install configparser elasticsearch statistics numpy pyyaml
+```
+
+You can then invoke cl as follows:
+
+```bash
+ python snafu/run_snafu.py -t cl
+```
+
+Note: additional arguments that can be passed are:
+
```

Comment:
Formatting issue with the below.

Reply:
seems to be [fixed](https://github.com/cloud-bulldozer/snafu/blob/656adc98e8068bdf7cc06874d3600b4f2096894c/cluster_loader/README.md)